specification/common-types/resource-management/v3/managedidentity.json

```diff
@@ -1,113 +1,158 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "3.0",
-    "title": "Common types"
+    "title": "Common types",
+    "version": "v3"
   },
```

Comment:
I feel like we first have a separate PR which only does "tsp-client sort" and merge. Then we do this PR.

Reply:
The command is here https://azure.github.io/typespec-azure/docs/migrate-swagger/01-get-started/

Reply:
thanks Pan! https://github.com/Azure/azure-rest-api-specs/pull/32574